README.md

```diff
@@ -1,8 +1,44 @@
-# Typst-Canvas
-A simple [Typst](https://typst.app) library for drawing stuff.
+# Canvas
 
-For more information, see the [manual](manual.pdf).
+Canvas is a simple [Typst](https://typst.app) library for drawing stuff.
 
-## Demos
-### Chart - [PDF](demo/3d-diagram.pdf) - [TYP](demo/3d-diagram.typ)
-[![Bar Chart](demo/3d-diagram.pdf.png)](demo/3d-diagram.typ)
+## Usage
+
+For information, see the [manual](manual.pdf).
+
+### Just
+
+This project uses [just](https://github.com/casey/just), a handy command runner.
+
+You can run all commands without having `just` installed, just have a look into the `justfile`.
+To install `just` on your system, use your systems package manager. On Windows, [Chocolatey](https://chocolatey.org/) (`choco install just`) can be used.
+
+## Installing
+
+To install the canvas package under your local typst package dir you can use the `install` script from the repository.
+
+```bash
+just install
+```
+
+The installed version can be imported by prefixing the package name with `@local`.
+
+```typ
+#import "@local/canvas:0.0.1"
+
+#canvas.canvas({
+  import canvas.draw: *
+  // Your drawing...
+})
+```
+
+## Testing
+
+This package comes with some unit tests under the `tests` directory.
+To run all tests you can run the `just test` target.
+
+You need to have ImageMagick installed on your system, which is needed for image comparison.
+
+### Windows
+If you are using the [Chocolatey](https://chocolatey.org/) package manager, you can install imagemagick using `choco install imagemagick`.
+Otherwise download and install a matching package from the [ImageMagick](https://imagemagick.org/script/download.php) website.
```

Comment:
Maybe just include a link to their main site like you've done with `just` and give a short description of what its used for.

Reply:
For ImageMagick? The reason its needed is in the prev. paragraph.